app/views/mission_control/jobs/jobs/_filters.html.erb

```diff
@@ -12,6 +12,16 @@
           <%= form.text_field :queue_name, value: @job_filters[:queue_name], class: "input", list: "queue-names", placeholder: "Filter by queue name..." %>
         </div>
 
+        <% if jobs_status == 'finished' %>
+          <div class="select is-rounded">
+            <%= form.datetime_field :finished_at_start, value: params.dig(:filter, :finished_at_start), class: "input", placeholder: "Finished from" %>
```

Comment:
I'd do the same as with the other parameters to get the value, rely on the already set `@job_filters`
```
@job_filters[:finished_at].begin
```
And below: 
```
@job_filters[:finished_at].end
```